sessionctx/variable/varsutil_test.go

```diff
@@ -74,6 +75,17 @@ func (s *testVarsutilSuite) TestNewSessionVars(c *C) {
 	c.Assert(vars.MemQuotaIndexLookupReader, Equals, int64(DefTiDBMemQuotaIndexLookupReader))
 	c.Assert(vars.MemQuotaIndexLookupJoin, Equals, int64(DefTiDBMemQuotaIndexLookupJoin))
 	c.Assert(vars.MemQuotaNestedLoopApply, Equals, int64(DefTiDBMemQuotaNestedLoopApply))
+
+	assertFieldsGreaterThanZero(c, reflect.ValueOf(vars.Concurrency))
+	assertFieldsGreaterThanZero(c, reflect.ValueOf(vars.MemQuota))
```

Comment:
memory quota can be smaller than 0, which means the memory usage is not limited

Reply:
It only verifies that the return value of `NewSessionVars`.